qiskit/transpiler/passes/basis/basis_translator.py

```diff
@@ -391,3 +440,20 @@ def _compose_transforms(basis_transforms, source_basis, source_dag):
                 )
 
     return mapped_instrs
+
+
+def _compute_incomplete_basis(target):
+    incomplete_basis_gates = []
+    size_dict = defaultdict(lambda: 0)
```

Comment:
Completely minor, but `int()` produces 0 as well, so `defaultdict(int)` is the docs-supported way of implementing a default counter.

Reply:
I updated this in the current version (hopefully backendv2 merges soon so I don't have to keep force pushing to update things and we can track dev history here separately).